CONTRIBUTING.md

```diff
@@ -1,28 +1,25 @@
 # Contributing
 
-We love contributions from everyone. By participating in this project, you
-agree to abide by the [thoughtbot code of conduct][tb-coc].
+We love contributions from everyone. By participating in this project, you agree to abide by the [thoughtbot code of
```

Comment:
Echoing [Joe's comment](https://github.com/thoughtbot/guides/pull/618#issuecomment-654188058) about 80 char line breaks. I learned that if you use VS Code there's a command palette option to auto-format this!

Reply:
I use `vim`, and I have an autoformatter that _should_ do this. It turns out that at some point, Prettier started fighting with it which was causing some pretty inconsistent behavior 😬 

I have normalized them to 80 characters now 🙂

Reply:
https://i.giphy.com/media/qN7NZR3Q5R2mY/giphy.webp